tech.json

```diff
@@ -51,6 +51,21 @@
               "name": "canCrossRoomJumpIntoWater",
               "requires": [ "canSuitlessMaridia" ],
               "note": "Using the momentum from jumping through a doorway from normal to water physics."
+            },
+            {
+              "name": "canSpaceJumpWaterBounce",
+              "requires": [
+                "canSuitlessMaridia",
+                "SpaceJump"
+              ],
+              "note": "Using Space Jump to bounce along the water line. It can be used to escape the water with a wall jump, or to get onto low platforms just above the water level.",
+              "extensionTechs": [
+                {
+                  "name": "canSpaceJumpWaterEscape",
+                  "requires": [ "canSpaceJumpWaterBounce" ],
+                  "note": "Using Space Jump with precise timing at the water line to escape water without Gravity Suit."
```

Comment:
I'd like a mention of `Space Jump Water Bounce` here.
Converting Space Jump Water Bouncing into regular space jumps without Gravity.  Often paired with HiJump.